mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMessageManager.java

```diff
@@ -467,6 +465,13 @@ protected MailboxMessage createMessage(Date internalDate, int size, int bodyStar
         return new SimpleMailboxMessage(messageIdFactory.generate(), internalDate, size, bodyStartOctet, content, flags, propertyBuilder, getMailboxEntity().getMailboxId(), attachments);
     }
 
+    protected MailboxMessage copyMessage(MailboxMessage message) throws MailboxException {
+        SimpleMailboxMessage copy = SimpleMailboxMessage.copyWithoutAttachments(message.getMailboxId(), message);
```

Comment:
withoutAttachments just here? Sounds like a hidden (un)capability

Reply:
That is. I'll imporove that and avoid leaking the "copyMessage knowledge" to sub classes...